Makefile

```diff
@@ -1,32 +1,35 @@
 DOCKER := $(shell which docker)
 DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.28.1
+PROJECTNAME=$(shell basename "$(PWD)")
 
-## proto-gen-docker: Generate protobuf files. Requires docker.
-proto-gen:
+.PHONY: help
+help: Makefile ## Display all available make commands
```

Comment:
This isn't working as expected

```
$ make help
 Choose a command to run in go-square:
sed: 1: "/^.PHONY: /{N; s/^.PHON ...": extra characters at the end of p command
```

Reply:
Pushed a fix...

```shell
$ make help
 Choose a command run in go-square:
  help         Get more info on make commands.
  proto-gen    Generate protobuf files. Requires docker.
  proto-lint   Lint protobuf files. Requires docker.
  lint         Lint Go files. Requires golangci-lint.
  test         Run unit tests.
  benchmark    Run tests in benchmark mode.
```

Reply:
That's interesting! May i know which env you are on ? I tested it on Pop_os though ..

Reply:
oh strange, maybe our `sed`s are incompatible? I'm on MacOS arm64. Unfortunately my sed doesn't tell me which version it is:

```
$ sed --version
sed: illegal option -- -
usage: sed script [-Ealnru] [-i extension] [file ...]
	sed [-Ealnu] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Any chance you can test the latest version of this PR on your machine?

Reply:
Now that you mention it, I remember something may be weird with MacOS `sed` because I hit this: https://github.com/celestiaorg/celestia-app/blob/5476636002d4c4be029718230d5f8f6d321e3b6f/scripts/single-node.sh#L64-L67

Reply:
Mine is 4.8. Recent change is working fine for me too.

Reply:
Okay awesome, thanks!